client/jest.config.js

```diff
@@ -0,0 +1,52 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: [
+    '**/src/components/**/*.{js}',
+    '!**/src/components/**/*{mock,stories,style,index}.{js}',
+    '!**/node_modules/**',
+    '!**/public/**',
```

Comment:
```suggestion
    '!**/build/**',
```
If this is meant for build (production) generated code base, then i think we are generating with the folder name as "build" instead of "public". :)